src/linear_combination.jl

```diff
@@ -1,72 +1,27 @@
 """
     LinearCombination(coefficients) <: Transform
 
-Calculate the linear combination using the vector coefficients passed in.
-"""
-struct LinearCombination <: Transform
-    coefficients::Vector{Real}
-end
-
-cardinality(::LinearCombination) = ManyToOne()
+Calculates the linear combination of a collection of terms weighted be some `coefficients`.
 
-"""
-    apply(
-        ::AbstractArray{<:Real, N}, ::LinearCombination; dims=1, inds=:
-    ) -> AbstractArray{<:Real, N-1}
-
-Applies the [`LinearCombination`](@ref) to each of the specified indices in the N-dimensional
-array `A`, reducing along the `dim` provided. The result is an (N-1)-dimensional array.
-
-The default behaviour reduces along the column dimension.
+When applied to an N-dimensional array, `LinearCombination` reduces along the `dim` provided
```

Comment:
I think we should add that the `dim` shouldn't be `:` (something to that effect). Currently this is catched by the deprecation warning, but we'll still need to handle `dims=:` once that is removed.